tools/fab/frontend-install.py

```diff
@@ -61,18 +61,13 @@ def banner(message):
 	print('#######################################')
 
 
-def copy(source, dest):
-	isodir = tempfile.tempdir()
-	banner("Copying %s to local disk" % source)
-	mount(source, isodir)
-	subprocess.call(['mkdir', '-p', dest])
-	subprocess.call(['cp', '-r', isodir, dest])
-	umount(isodir)
-
-
 def mount(source, dest):
 	subprocess.call(['mkdir', '-p', dest])
-	subprocess.call(['mount', '-o', 'loop,ro', source, dest])
+	proc = subprocess.Popen(['mount', '-o', 'loop,ro', source, dest], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
This should use `subprocess.run` instead:
```
result = subprocess.run(['mount', '-o', 'loop,ro', source, dest], capture_output = True)
if result.returncode != 0:
    result = subprocess.run('mount | grep iso', shell = True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
    print(result.stdout)
```

Reply:
It can't, as frontend-install.py needs to run against python 2.7

Reply:
Gross.